Sprint-1/implement/max.test.js

```diff
@@ -16,28 +16,51 @@ const findMax = require("./max.js");
 // When passed to the max function
 // Then it should return -Infinity
 // Delete this test.todo and replace it with a test.
-test.todo("given an empty array, returns -Infinity");
+test("an empty array should return -Infinity", () => {
+  expect(findMax([])).toBe("-Infinity");
+});
 
 // Given an array with one number
 // When passed to the max function
 // Then it should return that number
+test("an array with one number should return the number", () => {
+  expect(findMax([0])).toEqual(0);
+  expect(findMax([-4])).toEqual(-4);
+  expect(findMax([102])).toEqual(102);
+});
 
 // Given an array with both positive and negative numbers
 // When passed to the max function
 // Then it should return the largest number overall
+test("should return the largest number", () => {
+  expect(findMax([-4, -12, 0, -100, 1, 60, 112])).toEqual(112);
```

Comment:
This test is a good one.  However, when testing it is good practice to check what are called the "boundary" cases.  In this context you could have a test where the largest value is at index `0`, another test where the largest is at `list.length - 1` and one further test when the largest value is somewhere in the middle.